src/components/Upload.vue

```diff
@@ -0,0 +1,57 @@
+<template>
+  <UiLoading v-if="loading" />
+  <label v-else class="file-select">
+    <input type="file" @change="handleFileChange" accept="image/*, video/mp4" />
+    {{ $t('upload') }}
```

Comment:
It's better to use `<slot/>` to keep this component generic, later we will have the avatar displayed and upload button will be the avatar.